client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -199,8 +199,7 @@
                         <oui-action-menu
                             data-compact data-align="end"
                             data-ng-if="!$ctrl.isRetrievingSSLCertificate && ($ctrl.canOrderSSLCertificate() || $ctrl.canRegenerateSSLCertificate() || $ctrl.canDeleteSSLCertificate())">
-                            <oui-action-menu-item data-ng-if="$ctrl.canOrderSSLCertificate()"
-                                data-text="{{ ::'hosting_dashboard_service_order_ssl' | translate }}"
+                            <oui-action-menu-item data-text="{{ ::'hosting_dashboard_service_order_ssl' | translate }}"
```

Comment:
What's the point of `$ctrl.canOrderSSLCertificate()` if it doesn't determine when an user can order a certificate ? Are you sure you can order a certificate all the time ?

Reply:
Yes Button should be available everytime was the requirement since user can order a new certificate before the current one expires and also covers a use case to upgrade existing certificate.

Reply:
So shouldn't `canOrderSSLCertificate` be deleted ?

Reply:
Thanks @varun257 for the update!